client/src/pages/Home/Home.tsx

```diff
@@ -1,21 +1,50 @@
+import React, { useEffect, useState } from 'react';
 import styles from './Home.module.scss';
-import { Navbar } from '@/components';
 import { useGetUserQuery, selectLocation } from '@/store';
 import { useSelector } from 'react-redux';
 import { useLocation } from '@/hooks';
+import { Header, Map } from '@/components';
 
 export default function Home(): JSX.Element {
   const { data } = useGetUserQuery();
-  console.log(data);
+  const location = useLocation();
```

Comment:
useLocation doesn't return anything it simply dispatches an action that will retrieve the current location and store it inside the app state. So you don't need to assign it to a variable everything about that location will be obtained through the selector below this line

Reply:
Understood, I have removed the assignment. Thank you.